ando/tools/generator/utils.py

```diff
@@ -0,0 +1,86 @@
+import copy
+from pathlib import Path
+import pandas as pd
+import json
+
+
+def save_tsv(dataframe, path_to_save):
+    """
+    Append or create a tsv file corresponding of dataframe data
```

Comment:
I would expect that an empty line here is needed between the short description and the Parameters for the layout to be displayed properly in the documentation. @Slowblitz can you check that and fix it if needed?

Reply:
yep you are right